src/routes/controllers/articles.controller.js

```diff
@@ -50,9 +51,54 @@ export default async function create(req, res) {
       res,
       201,
       'your article has been created successfully',
-      createArticle,
+      createdArticle,
     );
   } catch (error) {
-    return errorResponse(res, 401, error.message);
+    return errorResponse(res, 500, error.message);
+  }
+}
+
+/**
+ * @description users can delete articles there have created.
+ * @param {object} req
+ * @param {object} res
+ * @returns {object} response object
+ */
+export async function removeArticle(req, res) {
+  try {
+    const userInfo = await verifyToken(
+      req.headers['x-access-token'] || req.headers.authorization,
+    );
+    const getArticle = await Article.findOne({
+      where: { id: req.params.article_id },
+    });
+
+    if (getArticle) {
+      if (getArticle.userId === userInfo.id) {
+        const deletedArticle = await Article.destroy({
+          where: { id: req.params.article_id },
+        });
+        if (deletedArticle > 0) {
+          return successResponse(
+            res,
+            200,
+            'article as been deleted successfully',
+          );
+        }
+        return errorResponse(res, 404, 'article does not exist');
+      }
+      return errorResponse(
+        res,
+        401,
+        'you are not authorized to perform this action',
+      );
+    }
+    return errorResponse(res, 404, 'article not found');
+  } catch (err) {
+    console.log(err.message, req.params);
```

Comment:
Why do we have a console log statement here?

Reply:
nice catch. thanks.